app/models/concerns/deactivable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'active_support/concern'
+
+module Deactivable
```

Comment:
I love the use of concern here, since we can use the same concern for other models in the future